teuthology/task/kernel.py

```diff
@@ -905,13 +906,15 @@ def grub2_kernel_select_generic(remote, newversion, ostype):
         mkconfig = 'grub-mkconfig'
     remote.run(args=['sudo', mkconfig, '-o', grubconfig, ])
     grub2conf = teuthology.get_file(remote, grubconfig, True)
+    log.debug('grub2conf is:\n%s' % grub2conf)
     entry_num = 0
     for line in grub2conf.split('\n'):
         if line.startswith('menuentry'):
             if newversion in line:
                 break
             entry_num += 1
     remote.run(args=['sudo', grubset, str(entry_num), ])
+    remote.run(args=['sudo', 'cat', '/boot/grub2/grub.cfg'])
```

Comment:
do we need ignore status flag here or is it fine?

Reply:
debugging right now, will drop thsi patch later